RecoTracker/LSTCore/src/alpaka/Event.h

```diff
@@ -0,0 +1,209 @@
+#ifndef RecoTracker_LSTCore_src_alpaka_Event_h
+#define RecoTracker_LSTCore_src_alpaka_Event_h
+
+#include "RecoTracker/LSTCore/interface/alpaka/Constants.h"
+#include "RecoTracker/LSTCore/interface/alpaka/LST.h"
+#include "RecoTracker/LSTCore/interface/Module.h"
+
+#include "Hit.h"
+#include "Segment.h"
+#include "Triplet.h"
+#include "Kernels.h"
+#include "Quintuplet.h"
+#include "MiniDoublet.h"
+#include "PixelQuintuplet.h"
+#include "PixelTriplet.h"
+#include "TrackCandidate.h"
+
+#include "HeterogeneousCore/AlpakaInterface/interface/host.h"
+
+using ::lst::EndcapGeometryBuffer;
+using ::lst::LSTESData;
+using ::lst::ModulesBuffer;
+using ::lst::PixelMap;
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+  namespace lst {
+
+    class Event {
+    private:
+      Queue queue;
+      Device devAcc;
+      DevHost devHost;
+      bool addObjects;
```

Comment:
[trivial]

Here and below, all data members should follow the usual coding rules, so
```suggestion
      Queue queue_;
      Device devAcc_;
      DevHost devHost_;
      bool addObjects_;
```